test/helpers/dashboards_helper_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class DashboardsHelperTest < ActionView::TestCase
```

Comment:
We should test the helper methods separately in the pregnancies model too! Adding an issue -- not related to this, just making a note that this is a pattern we should follow.